app/data_processing/data_initialization.py

```diff
@@ -0,0 +1,21 @@
+import json
+
+from app.structures.customer_profile import Car, Customer
+from app.structures.shop import Shop
+
+
+def get_data(filename: str) -> dict:
+    with open(filename, "r") as file:
+        data_dict = json.load(file)
+    return data_dict
+
+
+def create_account(customer: dict) -> Customer:
+    client = Customer(**customer)
+    client.car = Car(client.car["brand"], client.car["fuel_consumption"])

```

Comment:
Ensure that the keys in `client.car` dictionary match the parameters expected by the `Car` constructor. The keys should be `brand` and `fuel_consumption` as per the `Car` class definition.